install_config/persistent_storage/pod_security_context.adoc

```diff
@@ -158,116 +410,155 @@ Currently the list of volumes which support ownership management includes:
+
+Getting back to the link:#nfs-example[NFS example], the container needs it's UID set to 99,
+which is shown in the pod fragement above. Assuming the _default_ project and the _restricted_
+SCC, the pod's requested user id of 99 will *not* be allowed, and therefore the pod will fail.
+The pod fails because:
+
+- it requests 99 as its user id,
+- since all available SCCs use _MustRunAsRange_ for their `*runAsUser*` strategy, uid range
+checking is required, 
+- 99 is not included in the SCC or in namespace's user id range, so the pod fails.
+
+To fix this situation:
+
+- the _restricted_ SCC could be modified to include 99 within its min and max user id range
+(*not* recommended),
+- the _restricted_ SCC could be modified to use _RunAsAny_ for the `*runAsUser*` value,
+thus eliminating id range checking (*really not* recommended, containers could run as root),
+- a new SCC could be created with the appropriate user id range (recommended),
+- the _default_ project's UID range could be changed to allow a user id of 99.
+(not generally advisable since only a single range of user ids can be specified),
+- a new project could be created with the appropriate user id range defined (not covered here).
+
+[[scc-runasuser]]
+==== User Ids and Custom SCCs
+It's generally considered a good practice to *not* modify the predefined SCCs. The preferred approach
+is to create a custom SCC that better fits an organization's security needs, or create a new project
+that supports the desired user ids. Or, see
+link:../../dev_guide/projects.html#create-a-project[projects] to create a new project.
+
+A custom SCC can be created such that a min and max user id is defined, UID range
+checking is still enforced, and the UID of 99 will be allowed. Here is an example:
+```
+# oc export scc nfs-scc 
+allowHostDirVolumePlugin: false  #the allow* bools are the same as for the restricted scc
+...
+kind: SecurityContextConstraints
+metadata:
+  ...
+  name: nfs-scc <1>
+priority: 9 <2>
+requiredDropCapabilities: null
+runAsUser:
+  type: MustRunAsRange <3>
+  uidRangeMax: 99 <4>
+  uidRangeMin: 99
+...
+```
+<1> the name of this new SCC is "nfs-scc"
+<2> numerically larger numbers have greater priority, nil or omitted is the lowest priority.
+Higher priority SCCs sort before lower pri SCCs and thus have a better chance of matching a new pod.
+<3> the `runAsUser` strategy is set to _MustRunAsRange_, which means uid range checking is  enforced.
+<4> the uid range is 99-99 (a range of one value).
+
+Now, with `runAsUser: 99`, shown in the pod fragment above, the pod matches the new nfs-scc and is
+able to run with a UID of 99.
+
+[[selinux]]
 == SELinuxOptions
-
-The pod security context allows you to specify SELinux labels with which to run
-containers in your pod. Additionally, volumes which support SELinux management
-will be relabeled so that they are accessible by the specified label and,
+All predefined SCCs, except for the _privileged_ SCC, set the `seLinuxContext`
```

Comment:
Should we mention that selinux relabeling is only supported for non-shared devices?